tests/cpu/compare_with_acts_seeding.cpp

```diff
@@ -12,9 +12,9 @@
 #include "traccc/io/writer.hpp"
 
 // algorithms
-#include "traccc/clusterization/clusterization_algorithm.hpp"
+#include "traccc/seeding/seed_finding.hpp"
+#include "traccc/seeding/spacepoint_binning.hpp"
```

Comment:
I was always a bit confused by `seeding_algorithm` to be honest. Are we moving away from it? :thinking:

Reply:
I removed `traccc::algorithm` dependency from `test` directory, so I could not use `seeding_algorithm` here anymoew